### DISCUSSION TITLE
Ideas on Adding a Front End to the Application

### DISCUSSION THREAD

**Discussion by CodeAsherPlayz** [PythonMathCalculator/python-math-calculator#10] [Ideas]
This is a conversation that ha been suggested by Issue #9 by @rutvikpadhiyar000.

Issue Context:

1. Adding a front page to this Application
2. Actually making this into an application. 

@CodeAsherPlayz Ideas:

As the Owner of an HTML and CSS repository, I feel that I should add this as a function on the website that I have built. You may go to my user repository pages and find out about it. Or, we can turn this into a one-stop application instead.

General Comments:

No one in this repository is allowed to edit or delete this thead.

**Comment by ghost** [comment]
I don't think about taking it too far either but i want to try backend programming.

**Comment by ghost** [comment]
I too am new to programming i can make basic html pages but i am no good at designing so if you know html-css i may upload a basic version of website and you can do editing and css.

**Reply by CodeAsherPlayz** [reply]
Okay you can refer to my HTML / CSS repository and see what we can do.
